### PR TITLE
@eessex => Added video caption

### PR DESCRIPTION
**Pull request by dblock**
Added captions to video.

### CODE REVIEW

**Comment by eessex** [comment]
The caption itself is tested in the rich_text_component, but we'd want to update the video section's tests as well to make sure it is rendering/previewing correctly: https://github.com/artsy/positron/blob/master/client/apps/edit/components/section_video/test/index.coffee

The image component does a similar thing with the same component: https://github.com/artsy/positron/blob/master/client/apps/edit/components/section_image/test/index.coffee

File: client/apps/edit/components/section_video/test/index.coffee [hunk 1]
```diff
@@ -22,6 +22,7 @@ describe 'SectionVideo', ->
       props = {
```
Comment (eessex) [inline client/apps/edit/components/section_video/test/index.coffee]: You'll need to require the RichTextComponent with icons to get this test running - - see a working example here: https://github.com/artsy/positron/blob/master/client/apps/edit/components/section_image/test/index.coffee#L29

Comment (eessex) [inline client/apps/edit/components/section_video/test/index.coffee]: Additionally you will notice the [global window](https://github.com/artsy/positron/blob/master/client/apps/edit/components/section_image/test/index.coffee#L24) required just above on ln 24 and 25-- draft.js needs these declared in the test environment to run correctly

**Comment by eessex** [comment]
@kanaabe brought up the concept that maybe this field should be required -- if it is not filled out on articles with more than one video, we will still have the same problem of duplicate sitemap results.

**Comment by dblock** [comment]
Maybe we can make it required in a subsequent iteration? We should first go back and add descriptions to existing videos.

**Comment by dblock** [comment]
I cannot get the component to re-render in this one test:

```js
  it 'previews captions on keyup', ->
    @component.onCaptionChange('<p>updated</p>')
    $(ReactDOM.findDOMNode(@component)).html().should.containEql '<p>updated</p>'
```

But it's not a blocker.

File: client/apps/edit/components/section_video/test/index.coffee [hunk 2]
```diff
@@ -17,18 +17,20 @@ describe 'SectionVideo', ->
...
         section: new Backbone.Model
           url: 'https://www.youtube.com/watch?v=dQw4w9WgXcQ'
+          caption: '<p>Rick Astley - Never Gonna Give You Up</p>'
         editing: false
         setEditing: -> ->
       }
       @rendered = ReactDOMServer.renderToString React.createElement(SectionVideo, props)
       @component = ReactDOM.render React.createElement(SectionVideo, props), (@$el = $ "<div></div>")[0], => setTimeout =>
-        sinon.stub @component, 'setState'
```
Comment (dblock) [inline client/apps/edit/components/section_video/test/index.coffee]: This was 99% of my test problems, duh.

**Comment by dblock** [comment]
This is all good to go!

File: client/apps/edit/components/section_video/test/index.coffee (hunk 2 above)
Comment (eessex) [inline client/apps/edit/components/section_video/test/index.coffee]: Ah of course ¯\_(ツ)_/¯

**Comment by eessex** [comment]
Looks good, thanks for adding this! I'll take a look at requiring this field today.

**Pull request merged**